src/pbcontroller.cpp

```diff
@@ -113,9 +113,7 @@ bool PbController::setup_dirs_xdg(const char* env_home)
 		config_dir + NEWSBEUTER_PATH_SEP + config_file;
 
 	/* in data */
-	cache_file =
-		xdg_data_dir + NEWSBEUTER_PATH_SEP + cache_file;
-	lock_file = cache_file + LOCK_SUFFIX;
+	lock_file = xdg_data_dir + NEWSBEUTER_PATH_SEP + LOCK_SUFFIX;
```

Comment:
Am I reading this correctly that with XDG, Podboat locks ~/.local/share/newsboat/.lock (assuming XDG_DATA_DIR is is not set or is ~/.local/share), but with dotdirs, it locks ~/.newsboat/pb-lock.pid?

I understand you didn't make this change, I just wonder if I'm missing something here, and you probably read this code a bit more than I did.

Reply:
> Am I reading this correctly that with XDG, Podboat locks ~/.local/share/newsboat/.lock (assuming XDG_DATA_DIR is is not set or is ~/.local/share), but with dotdirs, it locks ~/.newsboat/pb-lock.pid?

You are right, definitely looks strange although it's not really a bug (the name is not mentioned in the documentation and it properly prevents a second podboat instance from starting).

Reply:
This is bizarre. Just to make sure that it's not a regression, I checked `git blame` and this code was so from the beginning: https://github.com/newsboat/newsboat/commit/404c19172ebeaca01a1aae8250c6820bc334d742 I think I'll go add this to the docs, because I'm sure I'll forget it at some point.